File: examples/Example_71.cs
using System;
using System.IO;
using System.Collections;
using System.Collections.Generic;
using System.Text;
using System.Diagnostics;

using PDFjet.NET;


/**
 *  Example_71.java
 *
 */
public class Example_71 {

    public Example_71() {

        PDF pdf = new PDF(new BufferedStream(
                new FileStream("Example_71.pdf", FileMode.Create)));

        Font f1 = new Font(pdf, new FileStream(
                "fonts/Droid/DroidSerif-Bold.ttf.stream",
                FileMode.Open,
                FileAccess.Read), Font.STREAM);
        f1.SetSize(12f);

        Font f2 = new Font(pdf, new FileStream(
                "fonts/Droid/DroidSerif-Italic.ttf.stream",
                FileMode.Open,
                FileAccess.Read), Font.STREAM);
        f2.SetSize(12f);

        Page page = new Page(pdf, Letter.PORTRAIT);

        CalendarMonth calendar = new CalendarMonth(f1, f2, 2018, 9);
        calendar.SetLocation(0f, 0f);
        float[] point = calendar.DrawOn(page);

	    CalendarMonth calendar2 = new CalendarMonth(f1, f2, 2018, 10);
        calendar2.SetLocation(0f, point[1]);
        calendar2.DrawOn(page);

        pdf.Complete();
    }


    public static void Main(String[] args) {
        Stopwatch sw = Stopwatch.StartNew();
        long time0 = sw.ElapsedMilliseconds;
        new Example_71();
        long time1 = sw.ElapsedMilliseconds;
        sw.Stop();
        Console.WriteLine("Example_71 => " + (time1 - time0));
    }

}   // End of Example_71.java

File: examples/Example_71.java
package examples;

import java.io.*;
import com.pdfjet.*;


/**
 *  Example_71.java
 *
 */
public class Example_71 {

    public Example_71() throws Exception {

        PDF pdf = new PDF(
                new BufferedOutputStream(new FileOutputStream("Example_71.pdf")));

        Font f1 = new Font(
                pdf,
                getClass().getResourceAsStream("../fonts/Droid/DroidSerif-Bold.ttf.stream"),
                Font.STREAM);
        f1.setSize(12f);

        Font f2 = new Font(
                pdf,
                getClass().getResourceAsStream("../fonts/Droid/DroidSerif-Italic.ttf.stream"),
                Font.STREAM);
        f2.setSize(12f);

        Page page = new Page(pdf, Letter.PORTRAIT);

        CalendarMonth calendar = new CalendarMonth(f1, f2, 2018, 9);
        calendar.setLocation(0f, 0f);
        float[] point = calendar.drawOn(page);

        CalendarMonth calendar2 = new CalendarMonth(f1, f2, 2018, 10);
        calendar2.setLocation(0f, point[1]);
        calendar2.drawOn(page);

        pdf.complete();
    }


    public static void main(String[] args) throws Exception {
        long time0 = System.currentTimeMillis();
        new Example_71();
        long time1 = System.currentTimeMillis();
        System.out.println("Example_71 => " + (time1 - time0));
    }

}   // End of Example_71.java
